src/components/_molecules/ButtonDeleteUser/ButtonDeleteUser.tsx

```diff
@@ -0,0 +1,11 @@
+import { DeleteOutlined } from '@ant-design/icons'
+import { Button, Popconfirm } from 'antd'
+import { ButtonDeleteUserType } from './ButtonDeleteUserType'
+
+export const ButtonDeleteUser = ({ onConfirm }: ButtonDeleteUserType) => {
+  return (
+    <Popconfirm title="Are you sure?" onConfirm={onConfirm}>
+      <Button icon={<DeleteOutlined />} />
```

Comment:
I think we will be reused it in different places, so change the name of component